scripts/update/00-venv.sh

```diff
@@ -14,3 +14,12 @@ if [[ -d /home/${user}/.venv ]]; then
         systemctl try-restart $app
     done
 fi
+
+if [[ -d /opt/.venv ]]; then
+    envs=($(ls /opt/.venv))
+    for app in ${envs[@]}; do
+        if ! grep -q "#\!/opt/.venv/${app}/bin/python" /opt/.venv/${app}/bin/*; then
+            sed -i "s|#\!/.*/bin/python|#\!/opt/.venv/${app}/bin/python|g" /opt/.venv/${app}/bin/*
+        fi
+    done
```

Comment:
I have not tested this at all so just shooting things on the wall
```suggestion
if [[ -d /opt/.venv ]]; then
    envs=($(find /opt/.venv/* -maxdepth 0 -type d))
    for venvpath in ${envs[@]}; do
        if ! grep -q "#\!/opt/.venv/${app}/bin/python" ${venvpath}/bin/*; then
            sed -i "s|#\!/.*/bin/python|#\!${venvpath}/bin/python|g" ${venvpath}/bin/*
        fi
    done
```